### DISCUSSION TITLE
Hood VPN not connecting

### DISCUSSION THREAD

**Discussion by mahdisarlak** [vpnhood/VpnHood#168] [Q&A]
Recently VPN Hood no longer works in our country (Iran) and does not connect, neither on the computer, nor on the mobile

This problem is the same on all internet providers

**Comment by trudyhood** [comment]
Hi,
Did you try to connect to our Vpnhood Public Servers or a Private VpnHood Server?

**Comment by mahdisarlak** [comment]
Private VpnHood Server

**Comment by mahdisarlak** [comment]
I tried this with different servers. And I asked other friends, they all had the same problem

**Reply by trudyhood** [reply]
> I tried this with different servers. And I asked other friends, they all had the same problem

That's important for us. 

1. Are you sure your server or hosting provider's IPs are not blocked? 
2. Can you send us one of your client log that could not connect?
3. Do you know how many users were connected to your server?

If you don't like to post this information here, send them to us via email: support@vpnhood.com

**Comment by techjoyment** [comment]
Same issue. They can detect your protocol. I just sent logs via your Windows built-in app "Send Report".

**Reply by trudyhood** [reply]
Please let me know the bug report reference ID as we have many reports. I can not find any email with your id in our bug reports repo
[support@vpnhood.com](mailto:support@vpnhood.com)
